### ISSUE TITLE
how to setup the csrf token on the backend?

### ISSUE THREAD

**Issue by AlyShmahell** [Spianopolese/webpyux#8]
if possible, an example of how to generate/authenticate this token would be great.
so far, it's static in the example and set to "session_token".

I'm assuming it could just be a random string stored in a session manager on the backend, but I'm not sure how the frontend is managing it so I don't know how it's being used by the site as a whole.

**Comment by Spianopolese** [comment]
> I'm assuming it could just be a random string stored in a session manager on the backend

Yes is a random string generated by the backend for the current session. The login is done in with two successive requests. In the first login request to authenticate it asks for the random token which it uses in the second request. To avoid a CSFR attack, the back-end must check that the token of the second request (sent in the HEADER in the X-CSRF-Token parameter) is the same as the one it saved in the first request. If it is the same it can log the user and return the information.
